### ISSUE TITLE
Windows Render (button/ctrl-shift-K) fails because of backslashes

### ISSUE THREAD

**Issue by fuhrmanator** [quarto-dev/quarto-vscode#95]
Render used to work (I'm not sure when it broke). I'm on Windows and the problem is with backslashes in bash.

Render button (or CTRL-SHIFT-K) runs the following command, leaving me in a Windows command shell:

    $ cmd /C"quarto preview c:/Users/Cris/Documents/GitHub/log210-ndc-quarto/MDD.qmd --no-browser --no-watch-inputs"
    Microsoft Windows [Version 10.0.19044.2130]      
    (c) Microsoft Corporation. All rights reserved.
    C:\Users\Cris\Documents\GitHub\log210-ndc-quarto>

It's remedied by exiting the command shell, recalling the last command and manually changing the `/` to `\` in the command (path to the filename).

VSCode says I have Quarto extension v1.46.0.

**Comment by jjallaire** [comment]
We had another user report that Render did not work within Git Bash on Windows due to backslashes: https://github.com/quarto-dev/quarto-vscode/issues/84

I confirmed that using backslashes in Git Bash does indeed result in an error:

![Screen Shot 2022-10-13 at 4 21 46 PM](https://user-images.githubusercontent.com/104391/195703023-54686acd-a4c2-4679-bcc7-21ab3a09959f.png)

So I changed to use forward slashes (which are legal in Windows paths) for all terminals and confirmed that this worked.

However, this is puzzling because you also mentioned using bash! Looking more closely at your code though I see the problem. You are likely running an older version of Quarto which requires us to run via `cmd /C` and in _that_ case the forward slashes don't work (I can repro this). 

So if you update to Quarto v1.1.63 or later then we will no longer need to use `cmd /C` and the problem should be remedied. If it's not convenient for you to update, I've also released a new version of the extension (v1.47.0) that will refrain from using the forward slashes when running w/ Quarto < v1.1.163
